modules/server-node/prisma-sqlite/schema.prisma

```diff
@@ -195,6 +214,18 @@ model OnchainTransaction {
   gasLimit        String
   gasPrice        String
 
+  createdAt DateTime @default(now())
+
+  onchainTransaction   OnchainTransaction? @relation(fields: [onchainTransactionId], references: [id])
+  onchainTransactionId String
```

Comment:
Imho, I'm not a fan of this reverse relationship here. For the sake of minimization, can we keep it out?

Reply:
I don't think we should ever retrieve 'individual' `onchainTransactionAttempt`